src/ontology/mondo-ingest.Makefile

```diff
@@ -328,8 +328,13 @@ mondo-ordo-subclass: $(REPORTDIR)/mondo_ordo_unsupported_subclass.tsv
 reports/mirror_signature-mondo.tsv: tmp/mondo.owl
 	$(ROBOT) query -i $< --query ../sparql/classes.sparql $@
 
-reports/mirror_signature-%.tsv: component-download-%.owl
-	$(ROBOT) query -i $(TMPDIR)/$<.owl --query ../sparql/classes.sparql $@
+# todo: it'd be ideal if only component-download-%.owl was only needed for the 'else' clause
+reports/mirror_signature-%.tsv: component-download-%.owl $(COMPONENTSDIR)/%.db metadata/%.yml
```

Comment:
FYI: I changed `reports/mirror_signature-%.tsv` into an if/else. If NCIT, it runs the python script which solves the `robot` memory issue. Otherwise, it generates it the normal way using SPARQL/`robot`.